lms/templates/footer-edx-v3.html

```diff
@@ -0,0 +1,91 @@
+## mako
+<%! from django.core.urlresolvers import reverse %>
+<%! from django.utils.translation import ugettext as _ %>
+<%namespace name='static' file='static_content.html'/>
+## WARNING: These files are specific to edx.org and are not used in installations outside of that domain. Open edX users will want to use the file "footer.html" for any changes or overrides.
+<footer id="footer-edx-v3" role="contentinfo" aria-label="Page Footer">
+    <h2 class="sr footer-about-title">${_("About edX")}</h2>
+
+    <div class="footer-logo">
+        <img alt="edX logo" src="${static.url('images/edx-theme/edx-header-logo.png')}">
+    </div>
+
+    <div class="site-details">
+        <nav class="site-nav" aria-label="About edX">
+            <a href="${marketing_link('ABOUT')}">${_("About")}</a>
+            <a href="${marketing_link('FAQ')}">${_("FAQs")}</a>
+            <a href="${marketing_link('NEWS')}">${_("News")}</a>
+            <a href="${marketing_link('BLOG')}">${_("Blog")}</a>
+            <a href="${marketing_link('CONTACT')}">${_("Contact")}</a>
+            <a href="${marketing_link('JOBS')}">${_("Jobs")}</a>
+            <a href="${marketing_link('DONATE')}">${_("Donate")}</a>
+            <a href="${marketing_link('DONATE')}">${_("Sitemap")}</a>
+        </nav>
+        <nav class="legal-notices" aria-label="Legal">
+            <a href="${marketing_link('TOS')}">${_("Terms of Service &amp; Honor Code")}</a>
+            <a href="${marketing_link('PRIVACY')}">${_("Privacy Policy")}</a>
+            <a href="${marketing_link('PRIVACY')}">${_("Website Accessibility Policy")}</a>
+        </nav>
+        <p class="copyright">
+            ## Using "edX Inc." explicitly here for copyright purposes (settings.PLATFORM_NAME is just "edX", and this footer is only used on edx.org)
+            ## Site operators: Please do not remove this paragraph! This attributes back to edX and makes your acknowledgement of edX's trademarks clear.
+            <p>&copy; ${settings.COPYRIGHT_YEAR} edX Inc. All rights reserved except where noted. EdX, Open edX and the edX and Open edX logos are registered trademarks of trademarks of edX Inc.</p>
```

Comment:
The data is actually going to come from an API so I think it shouldn't be. I will discuss with @wedaly tomorrow